pom.xml

```diff
@@ -410,7 +405,6 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
-                    <aggregate>true</aggregate>
```

Comment:
I don't know what this line is for, but if you removed it on purpose, you get my approval! 😄

Reply:
yes, it is apparently no longer a valid config property for the javadoc plugin as it was giving us a warning.